### ISSUE TITLE
the Interrogation button doesn't work.

### ISSUE THREAD

**Issue by AbdullahAlfaraj** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#270]
Does the interrogate button depend on any model?
              I have the tagger plugin?
              deepbooru and clip work fine in webui.
              but the interrogate button does not work in the ps plugin.

_Originally posted by @XiaoCC in https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/issues/266#issuecomment-1561093154_

**Comment by AbdullahAlfaraj** [comment]
@XiaoCC, the interrogation button doesn't depend on any SD model, but it uses the clip model.
![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/7842232/0d2f7056-de2a-46be-bed2-8614dad4535e)

visit http://127.0.0.1:7860/docs#/default/interrogateapi_sdapi_v1_interrogate_post
and paste the following in (2), see if you get similar response as in the screenshort (4)

```
{
  "image": "iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAIAAADTED8xAAADMElEQVR4nOzVwQnAIBQFQYXff81RUkQCOyDj1YOPnbXWPmeTRef+/3O/OyBjzh3CD95BfqICMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMO0TAAD//2Anhf4QtqobAAAAAElFTkSuQmCC",
  "model": "clip"
}
```

**Comment by XiaoCC** [comment]
<img width="1303" alt="image" src="https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/7299222/6ab20336-82ea-4375-92be-1f53e4091d3d">

**Comment by XiaoCC** [comment]
@AbdullahAlfaraj  
I am remote calling. sd-url
Could it be that the interrogate button only calls the local stable diffusion?

**Comment by AbdullahAlfaraj** [comment]
The issue has been resolved in version 1.2.6. It will be available to everyone once it’s released.
